x-pack/plugins/spaces/public/management/edit_space/edit_space_tabs.tsx

```diff
@@ -92,7 +93,11 @@ export const getTabs = ({
     },
   ];
 
-  if (canUserViewRoles) {
+  const canUserViewRoles = Boolean(capabilities?.roles?.view);
+  const isRoleManagementEnabled = authz && authz.isRoleManagementEnabled() !== false;
```

Comment:
```suggestion
  const isRoleManagementEnabled = authz && authz.isRoleManagementEnabled();
```

Reply:
@jeramysoucy I think we examined this further in Slack, and agreed that the `config.roleManagementEnabled` is only available in serverless, which makes the `authz.isRoleManagementEnabled()` return undefined in stateful, when it actually should be interpreted as being enabled.

So how about:
```
  /*
   * config.roleManagementEnabled is an offering-based config. In serverless, the value will be true or false.
   * In stateful, role management is enabled but the config setting is expected to be undefined.
   */
  const isRoleManagementEnabled =
    authz &&
    (typeof authz.isRoleManagementEnabled() === 'undefined' || authz.isRoleManagementEnabled());
```

Alternatively, we could update the implementation of `authz.isRoleManagementEnabled` in https://github.com/elastic/kibana/blob/4339f8465720c507351a2e65a6fe881a10dcd938/x-pack/plugins/security/public/authorization/authorization_service.ts#L21

Reply:
Moved this logic to x-pack/plugins/spaces/public/management/edit_space/provider/edit_space_provider.tsx

Reply:
Thanks, Tim! Sorry for the confusion.